config/unpack_archived_dataset.sh

```diff
@@ -3,19 +3,22 @@ set -ex
 echo -e '\n'
 echo "Check files processing on student dataset"
 
-TARGET_SCORE=$(bash config/get_mark.sh lab_6_pipeline)
+TARGET_SCORE_SCRAPPER=$(bash config/get_mark.sh lab_5_scrapper)
+TARGET_SCORE_PIPELINE=$(bash config/get_mark.sh lab_6_pipeline)
 
 
-if [[ ${TARGET_SCORE} != 0 ]]; then
+if [[ ${TARGET_SCORE_SCRAPPER} != 0 || ${TARGET_SCORE_PIPELINE} != 0 ]]; then
   mkdir -p tmp/articles
-  mv *_cleaned.txt tmp/articles
   mv *_raw.txt tmp/articles
+  mv *_cleaned.txt tmp/articles || echo "no files to move"
 
-  if [[ ${TARGET_SCORE} != 4 ]]; then
+  if [[ ${TARGET_SCORE_SCRAPPER} -gt 4 || ${TARGET_SCORE_PIPELINE} -gt 4 ]]; then
```

Comment:
I do not agree - why do we start caring about lab 5 in this script?

Reply:
was fixed by Vita